auth4genai/snippets/get-started/langchain-next-js/auth-for-rag.mdx

```diff
@@ -112,10 +112,10 @@ export const getContextDocumentsTool = tool(
 
 Call the tool from your AI agent to get data from documents. First, update the `/src/app/api/chat/[..._path]/route.ts` file with the following code to pass the user credentials to your agent:
 
-```ts src/app/api/chat/[..._path]/route.ts wrap lines highlight={3,19}
+```ts src/app/api/chat/[..._path]/route.ts wrap lines highlight={3,19,31}
 import { initApiPassthrough } from "langgraph-nextjs-api-passthrough";
 
-import { getUser } from "@/lib/auth0";
+import { getUser, getAccessToken } from "@/lib/auth0";
```

Comment:
i dont see a point of updating the rag quickstart as this change is not required for this quiclkstart to work at all. Its ok to update the sample app as good practice but I think the quickstart need not be updated. I'll leave the decision to you @KarimTantawy5